resources/js/Pages/Event.vue

```diff
@@ -125,16 +125,27 @@ const colorPicker = () => {
                 text-size="sm:text-3xl"
                 padding="sm:px-8"
                 @click="
-                    $page.props.auth.user
-                        ? isEnrolled
-                            ? router.put(route('event.join', event))
-                            : router.get(route('home') + '#enroll-section') // HACK: this is a hack
-                        : router.get(route('register'))
+                    if (isEnrolled && event.external_url) {
+                    } else {
+                        $page.props.auth.user
+                            ? isEnrolled
+                                ? router.put(route('event.join', event))
+                                : router.get(route('home') + '#enroll-section') // HACK: this is a hack
+                            : router.get(route('register'));
+                    }
```

Comment:
Is this "if" correct?

Reply:
I meant for the button to have no behavior in that case, so it's just a link essentially

Reply:
Looks very sketchy though

Reply:
Can't you do this instead? Looks better than an empty if statement imo
```suggestion
                    if (!(isEnrolled && event.external_url)) {
                        $page.props.auth.user
                            ? isEnrolled
                                ? router.put(route('event.join', event))
                                : router.get(route('home') + '#enroll-section') // HACK: this is a hack
                            : router.get(route('register'));
                    }
```

Reply:
yes that's much better, I don't know why I thought an empty if was a good idea :skull: